tools/src/main/java/com/nedap/archie/adl14/ADL14Converter.java

```diff
@@ -23,73 +21,53 @@
 public class ADL14Converter {
 
     private final MetaModels metaModels;
-    private TerminologyAccess terminologyAccess = OpenEHRTerminologyAccess.getInstance();
     private final ADL14ConversionConfiguration conversionConfiguration;
-    private InMemoryFullArchetypeRepository existingRepository;
 
     public ADL14Converter(MetaModels metaModels, ADL14ConversionConfiguration conversionConfiguration) {
         this.metaModels = metaModels;
         this.conversionConfiguration = conversionConfiguration;
     }
 
-
-    /**
-     * Set the base repository for this converter. If you don't set it yourself, the converter will create an empty repository.
-     * The archetypes in the repository will be used as base archetypes, but will not be converted themselves. Please
-     * note that this repository will be modified, it will have all the converted archetypes added to it!
-     *
-     * @param existingRepository the existing repository to use
-     */
-    public void setExistingRepository(InMemoryFullArchetypeRepository existingRepository) {
```

Comment:
I do not think it is not a good idea to remove this. It may be unused in Archie at the moment, but is necessary if you are converting OPTs, for example. It is part of the public API of Archie.

Reply:
It is used at least here, and without it, this functionality cannot be built: https://github.com/nedap/archetype-languageserver/blob/a379ff5c81760d0601a9d18589585cd5cbc6d844/src/main/java/com/nedap/openehr/lsp/repository/ADL14ConvertingStorage.java#L51